src/react/components/common/tagInput/tagInput.tsx

```diff
@@ -612,17 +612,29 @@ export class TagInput extends React.Component<ITagInputProps, ITagInputState> {
 
     private getTypeSubMenuItems = (): IContextualMenuItem[] => {
         const tag = this.state.selectedTag;
-        const types = Object.values(FieldType);
+        let types = Object.values(FieldType);
 
-        return types.map((type) => {
-            return {
-                key: type,
-                text: type,
+        if (tag.type === "checkbox") {
+
+            return [{
+                key: tag.type,
+                text: tag.type,
                 canCheck: true,
-                isChecked: type === tag.type,
+                isChecked: true,
                 onClick: this.onTypeSelect,
-            } as IContextualMenuItem;
-        });
+            } as IContextualMenuItem];
+        } else {
+            return types.filter((type) => type !== "checkbox")
```

Comment:
The filter removes user ability to change type of tag to checkbox manually, is this what we want?

Reply:
We want not allow user to change tag type to checkbox if it was assign to anything else than checkbox.

Reply:
But with this filter makes it so user can't assign tag to checkbox

Reply:
![image](https://user-images.githubusercontent.com/60453211/80635432-bb050680-8a10-11ea-9c0f-2bf86d6eb5a2.png)

Reply:
Yes, you are right. I'm working on it - we need create a flag on label interface, in order to know if it is a brand new label or it was assigned before. Right now it can infer checkbox type from ocr.